location/gql_mutations.py

```diff
@@ -263,6 +263,9 @@ class Input(HealthFacilityInputType):
     @classmethod
     def async_mutate(cls, user, **data):
         try:
+            if HealthFacilityService.check_unique_code(data['code']):
```

Comment:
Is it possible to get key error in here? If so please change 
```suggestion
            if HealthFacilityService.check_unique_code(data.get('code')):
```

I know this field is mandatory and creation would fail later on in process, but request shouldn't end with key error.